### DISCUSSION TITLE
How to enable port forwarding function

### DISCUSSION THREAD

**Discussion by weisiqian** [coder/coder#7691] [General] [answered]
![image](https://github.com/coder/coder/assets/19994951/c8ce9edd-d4d3-42fe-bb1f-251ad2d9f9a4)

I have configure the CODER_ACCESS_URL and CODER_WILDCARD_ACCESS_URL.

```
CODER_ACCESS_URL=http://172.28.216.159.nip.io:7080
CODER_WILDCARD_ACCESS_URL=*.172.28.216.159.nip.io:7080
```

**Comment by matifali** [comment, marked as answer]
The access URL and the wildcard URL shouldn't have the ports
They are the URLs that should point to your public IP.

**Reply by matifali** [reply]
See the related issue. #8189
